static/js/hotkey.js

```diff
@@ -596,7 +596,8 @@ exports.process_hotkey = function (e, hotkey) {
     }
 
     if (event_name === 'left_arrow') {
-        if (overlays.lightbox_open()) {
+        // Disables left_arrow key when 'Pan and zoom' is enabled
+        if (overlays.lightbox_open() && !$(".enabled")[0]) {
```

Comment:
What is that `.enabled` test supposed to do?  It seems way too generic to be correct.

Reply:
In `lightbox.js` in this part:-
```
    $("#lightbox_overlay").on("click", ".lightbox-canvas-trigger", function () {
        let $img = $("#lightbox_overlay").find(".image-preview img");

        if ($img.length) {
            $(this).addClass("enabled");
            // the `lightbox.open` function will see the enabled class and
            // enable the `LightboxCanvas` class.
            exports.open($img);
        } else {
            $img = $("#lightbox_overlay").find(".image-preview canvas")[0].image;

            $(this).removeClass("enabled");
            exports.open($img);
        }
    });
``` 
a class `enabled` is added to `lightbox-canvas-trigger` when "Pan and Zoom" is enabled. 

The `.enabled` test checks whether "Pan and Zoom" is enabled and if it is enabled, it changes the default behavior of the left and right arrow key of changing image. In `lightbox_canvas.js`, the behavior for the left and right arrow key is defined. In that file, it checks whether the image's zoom is 1 or more than 1. If it is equal to 1, the arrow keys have the current behavior of switching the images. If the zoom is greater than 1, the arrow keys will be used to pan around the image. Also the swich_image widget disappears as the zoom gets greater than 1.